docker/nginx/default.conf

```diff
@@ -4,7 +4,7 @@ server {
     listen 80 default_server;
 
     # Set the document root of the project
-    root /PATH/TO/YOUR/DOCUMENT/ROOT;
+    root /etp_web;
```

Comment:
This line was deliberately written as it was, and should also be drawn from environment variables, to be fair.